app/controllers/smart_proxies_controller.rb

```diff
@@ -108,4 +108,11 @@ def action_permission
         super
     end
   end
+
+  def proxy_resource_base
```

Comment:
I am okay with that, we do similar things around the codebase, this one is three lines but what can we do about it. @tbrisker ? Please ack and I am gonnat give it a whirl.